packages/exo/src/exo-tools.js

```diff
@@ -6,59 +6,156 @@ import {
   mustMatch,
   M,
   isAwaitArgGuard,
+  isRawGuard,
   getAwaitArgGuardPayload,
   getMethodGuardPayload,
   getInterfaceGuardPayload,
   getCopyMapEntries,
 } from '@endo/patterns';
 
+import { GET_INTERFACE_GUARD } from './get-interface.js';
+
 /** @typedef {import('@endo/patterns').Method} Method */
 /** @typedef {import('@endo/patterns').MethodGuard} MethodGuard */
+/** @typedef {import('@endo/patterns').MethodGuardPayload} MethodGuardPayload */
 /**
  * @template {Record<PropertyKey, MethodGuard>} [T=Record<PropertyKey, MethodGuard>]
  * @typedef {import('@endo/patterns').InterfaceGuard<T>} InterfaceGuard
  */
-/** @typedef {import('@endo/patterns').InterfaceGuardKit} InterfaceGuardKit */
 
 const { quote: q, Fail } = assert;
 const { apply, ownKeys } = Reflect;
 const { defineProperties, fromEntries } = Object;
 
+/**
+ * A method guard, for inclusion in an interface guard, that does not
+ * enforce any constraints of incoming arguments or return results.
+ */
+const RawMethodGuard = M.call().rest(M.raw()).returns(M.raw());
```

Comment:
not much of a _guard_  ;-)

consider `RawMethodCall` or `UnguardedCall`

Reply:
It is still a `MethodGuard`, so I don't want to obscure that with a different name.